docs/scripts/object_orientated_programming_poc/oop_poc_dataset.py

```diff
@@ -0,0 +1,76 @@
+import pandas as pd
```

Comment:
For the FileView I had considered making this script but I felt that jumping right in and making the tutorial was more effective. If you have time it'd be great if a tutorial could be created, otherwise this is sufficient for this PR.

https://synapsepythonclient--1181.org.readthedocs.build/en/1181/tutorials/python/fileview/

Reply:
I'll go ahead and add this!

Reply:
[added](https://github.com/Sage-Bionetworks/synapsePythonClient/pull/1175/commits/ff4d0c69199f06d257db95c73781f952520d6a40)